app/routes/events/components/EventEditor/EditorSection/Header.tsx

```diff
@@ -0,0 +1,160 @@
+        required
+      />
+
+      <Modal
+        show={showImageGallery}
+        onHide={() => setShowImageGallery(false)}
+        contentClassName={styles.imageGallery}
+      >
+        <>
+          <h1>Bildegalleri</h1>
+          <Flex
+            wrap
+            alignItems="center"
+            justifyContent="space-around"
+            gap="1rem"
+          >
+            {imageGallery?.map((e) => (
+              <Flex key={e.key} alignItems="center" gap="1rem">
+                <Image
+                  src={e.cover}
+                  placeholder={e.coverPlaceholder}
+                  alt={`${e.cover} bilde`}
+                  onClick={() => {
+                    form.change('cover', `${e.key}:${e.token}`);
+                    setShowImageGallery(false);
+                    setUseImageGallery(true);
+                    setImageGalleryUrl(e.cover);
+                  }}
+                  className={styles.imageGalleryEntry}
+                />
+                <ConfirmModal
+                  title="Fjern fra galleri"
+                  message={`Er du sikker på at du vil fjerne bildet fra bildegalleriet? Bildet blir ikke slettet fra databasen.`}
+                  closeOnConfirm
+                  onConfirm={() => setSaveForUse(e.key, e.token, false)}
+                >
+                  {({ openConfirmModal }) => (
+                    <Icon onClick={openConfirmModal} name="trash" danger />
+                  )}
+                </ConfirmModal>
+              </Flex>
+            ))}
+            {imageGallery.length === 0 && (
+              <Flex
+                column
+                alignItems="center"
+                gap={5}
+                className={styles.emptyGallery}
+              >
+                <Icon name="folder-open-outline" size={50} />
+                <b>Bildegalleriet er tomt ...</b>
+                <span>Hvorfor ikke laste opp et bilde?</span>
+              </Flex>
+            )}
+          </Flex>
+        </>
+      </Modal>
+
+      <Flex
+        alignItems="center"
+        justifyContent="space-between"
+        margin={'0 0 1rem 0'}
+      >
+        <Button onClick={() => setShowImageGallery(true)}>
+          Velg bilde fra bildegalleriet
+        </Button>
+        <div>
+          <Field
+            label="Lagre til bildegalleriet"
+            description="Lagre bildet til bildegalleriet slik at det kan bli brukt til andre arrangementer"
+            name="saveToImageGallery"
+            type="checkbox"
+            component={CheckBox.Field}
+            fieldClassName={styles.metaField}
+            className={styles.formField}
+          />
+        </div>
+      </Flex>
+
+      <Field
+        name="youtubeUrl"
+        label="Erstatt cover-bildet med video fra YouTube"
+        description="Videoen erstatter ikke coveret i listen over arrangementer"
+        placeholder="https://www.youtube.com/watch?v=bLHL75H_VEM&t=5"
+        component={TextInput.Field}
+      />
```

Comment:
```suggestion
      <Field
        name="youtubeUrl"
        label="Erstatt cover-bildet med video fra YouTube"
        description="Videoen erstatter ikke coveret i listen over arrangementer"
        placeholder="https://www.youtube.com/watch?v=bLHL75H_VEM&t=5"
        component={TextInput.Field}
        parse={(v) => !!v}
      />
```

If you want to, and since you kind of started, you can slap on this to resolve ABA-826

Reply:
I'm assuming you meant
```
parse={(v) => v}
```
?
I'd think `parse={(v) => !!v}` would just turn the youtube url into a boolean value, which would probably fix the bug, but would undoubtedly create a new one

Reply:
ah yes I meant (v) => v
I just copied an example I had in front of me not realizing it was wrong